me.spaceinbox.actioneer.yaml

```diff
@@ -0,0 +1,56 @@
+app-id: me.spaceinbox.actioneer
+runtime: org.gnome.Platform
+runtime-version: "49"
+# Use the GNOME SDK for matching runtime-version 49 (keep runtime up-to-date to avoid EOL)
```

Comment:
```suggestion
```

Reply:
A useful comment who have explained why you need GNOME vs Freedeskop-SDK so that one doesn't have to ask right now why you need the GNOME SDK.

Reply:
Reviewer note: it uses GTK4.